Tasks/SonarQubePostTest/PRCA/PostComments-Module.psm1

```diff
@@ -358,10 +266,7 @@ function FilterCommentsByPath
 #
 function FilterPreExistingComments
 {
-     param ([Array]$messages, 
-     [ValidateNotNullOrEmpty()][string]$messageSource, 
```

Comment:
During testing I realized that an empty array can be the same as $null for PowerShell so I removed the Validate Not Null attributes on methods that can potentially receive empty arrays.